src/test/java/io/orkes/conductor/client/LocalWorkerTest.java

```diff
@@ -25,7 +25,7 @@ public class LocalWorkerTest {
 
     public static void main(String[] args) {
         ApiClient apiClient = new ApiClient("http://localhost:8080/api");
-        apiClient.setUseGRPC("localhost", 8090);
+        //apiClient.setUseGRPC("localhost", 8090);
```

Comment:
Should we remove this comment? Or maybe add a note about it's purpose.

Reply:
done.